bundles/org.openhab.binding.weathercompany/README.md

```diff
@@ -0,0 +1,1040 @@
+| forcastDay(0-5)(Day\|Night)   | windDirectionCardinal           | String                  | Average wind direction in cardinal notation                    |
+| forcastDay(0-5)(Day\|Night)   | windPhrase                      | String                  | A phrase that describes the wind direction and speed           |
+| forcastDay(0-5)(Day\|Night)   | precipitationChance             | Number:Dimensionless    | Maximum probability of precipitation                           |
+| forcastDay(0-5)(Day\|Night)   | precipitationType               | String                  | Type of precipitation to display (e.g. rain, snow)             |
+| forcastDay(0-5)(Day\|Night)   | precipitationRain               | Number:Length           | The forecasted measurable liquid precipitation                 |
+| forcastDay(0-5)(Day\|Night)   | precipitationSnow               | Number:Length           | The forecasted measurable precipitation as snow                |
+| forcastDay(0-5)(Day\|Night)   | snowRange                       | String                  | Snow accumulation amount for the forecast period               |
+| forcastDay(0-5)(Day\|Night)   | thunderCategory                 | String                  | The description of probability of thunderstorm activity        |
+| forcastDay(0-5)(Day\|Night)   | thunderIndex                    | Number                  | An enumeration of thunderstorm probability                     |
+| forcastDay(0-5)(Day\|Night)   | uvDescription                   | String                  | Level of risk of skin damage due to exposure                   |
+| forcastDay(0-5)(Day\|Night)   | uvIndex                         | Number                  | Maximum UV index for the forecast period                       |
+| forcastDay(0-5)(Day\|Night)   | iconCode                        | Number                  | Key to the weather icon lookup                                 |
+| forcastDay(0-5)(Day\|Night)   | iconCodeExtend                  | String                  | Code representing full set sensible weather                    |
+| forcastDay(0-5)(Day\|Night)   | iconImage                       | Image                   | Image representing forecast condition                          |
+| forcastDay(0-5)(Day\|Night)   | qualifierPhrase                 | String                  | Describes special weather criteria                             |
+| forcastDay(0-5)(Day\|Night)   | qualifierCode                   | String                  | Code for special weather criteria                              |
+
+### Local Language Support
+
+The following channels will be translated to local language based on the language setting in the thing configuration.
+
+- dayOfWeek
+- daypartName
+- narrative
+- qualifierPhrase
+- uvDescription
+- windDirectionCardinal
+- windPhrase
+- wxPhraseLong
+
+ 
+## Full Example
+
+### Thing Example
+
+```
+Bridge weathercompany:bridge:bridge [ apiKey="734982347982374" ] {
```

Comment:
I hope the API key isn't a real one...

The nice thing with the account as a bridge is that you only need to declare it once:
```suggestion
Bridge weathercompany:bridge:bridge [ apiKey="734982347982374" ] {
    Thing weathercompany:weather-forecast:forecast "Local Forecast" @ "Home" [locationType="postalCode",postalCode="10001:US",language="en_US",refreshInterval=30]
    Thing weathercompany:weather-forecast:chitown "Chicago Forecast" @ "Ohare Airport" [apiKey="734982347982374",locationType="iataCode",iataCode="ORD",language="en_US",refreshInterval=30]    
    Thing weathercompany:weather-forecast:miami "Miami Weather" @ "South Beach" [locationType="postalCode",postalCode="33139:US",language="es_US",refreshInterval=30]
    Thing weathercompany:weather-observations:observations "Local Observations" @ "Home" [pwsStationId="KFLMIAMI208",refreshInterval=30]    
    Thing weathercompany:weather-observations:patagonia "Torres del Paine Weather" @ "Patagonia" [pwsStationId="IPUNTAAR4",refreshInterval=30]
}
```

Reply:
Good idea to combine together.